internal/graph/check.go

```diff
@@ -761,7 +770,7 @@ func (c *LocalChecker) checkDirectUsersetTuples(ctx context.Context, req *Resolv
 			}
 			defer iter.Stop()
 
-			if !c.optimizationsEnabled {
+			if !c.ff.Boolean(serverconfig.ExperimentalCheckOptimizations, false, nil) {
```

Comment:
In `Check`, we should retain `optimizationsEnabled` as a struct setting.

The inspection of the flag should happen right before calling _ResolveCheck_ at the controller. The reason being, we want a stable Check resolution. Don't want some nodes being checked with optimizations and others not in the case of a flip. Could lead to unstable results (or if the FF service is down/serving stale)